src/main/java/xyz/aadev/generitech/common/config/ConfigWorldGen.java

```diff
@@ -125,123 +123,123 @@ public static void syncConfig(Configuration config) {
         final String DESC_BIOME_RESTRICTION_TYPE = "Biome Restriction Type (Either 'blacklist' or 'whitelist')";
         final String DESC_BIOME_RESTRICTION_LIST = "Biome IDs to restrict ore spawning in";
 
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Rename this local variable name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.aadev.xyz/coding_rules#rule_key=squid%3AS00117)